### PR TITLE
Three additions/changes - 1 file for enzymatic reactions, 1 file for local efficiency and added LocalEfficiency()

### PR DESCRIPTION
**Pull request by iamargho**
**Made three changes:**

1. Added the LocalEfficiency() function that will compute local efficiency scores for the organism.
2. Added dossier_example_05_organism_local_efficiency.py (every measure will have a different file).
3. Added enzymatic genes dictionary and the respective tuples containing reactant-products (as a different file).

Kindly review.

### CODE REVIEW

File: dossier.py [hunk 1]
```diff
@@ -6,6 +6,9 @@
 Date created: 17th April 2021
 """
 
+from metabolism import *
```
Comment (mauriceling) [inline dossier.py:9]: metabolism should be import in  dossier_example_05_organism_local_efficiency.py 
not dossier itself

File: dossier.py [hunk 2]
```diff
@@ -619,3 +622,39 @@ def SubsequenceCounter(dataframe, replicate, kwargs):
...
+
+    @param dataframe: Returned dataframe from dossier.
+    DOSE_Result_Database.OrgParam_Time()
+    @param replicate: Replicate number
+    @type replicate: Integer
+    @param kwargs: Keyword parameters used for fitness calculation.
+    @return: [Replicate, Generation, DO(1), ..., DO(n)] of fitness 
+    scores.
+    """
+    subsequence = kwargs["subsequence"]
```
Comment (mauriceling) [inline dossier.py:639]: add
enzymatic_reactions = kwargs["enzymatic_reactions"]

File: dossier.py [hunk 3]
```diff
@@ -619,3 +622,39 @@ def SubsequenceCounter(dataframe, replicate, kwargs):
...
+    generations = list(set(dataframe["generation"].tolist()))
+    generations.sort()
+    fitnessTable = []
+    def _core(sequence):
```
Comment (mauriceling) [inline dossier.py:643]: should be 
def _core(sequence, enzymatic_reactions):

**Review by mauriceling — CHANGES_REQUESTED** [review]
metabolism should be imported in dossier_example_05_organism_local_efficiency.py 
not in dossier.py
Now, you are requiring everybody to have metabolism.py

File: dossier.py (hunk 1 above)
Comment (iamargho) [reply dossier.py:9]: Okay understood.

**Comment by iamargho** [comment]
Made requested changes.

Please review.

**Pull request merged**